src/Vat.sol

```diff
@@ -84,24 +84,23 @@ contract Vat {
     }
 
     // --- Math ---
+    string private constant ARITHMETIC_ERROR = string(abi.encodeWithSignature("Panic(uint256)", 0x11));
```

Comment:
Any reason to switch to this error way? I mean I'm not against it but not sure if this is something we want to take as new standard and that case we would need some discussion. I would go first with the standard `Vat/overflow` error message.

Reply:
So we've already introduced a switch to the default arithmetic error almost everywhere by upgrading to Solidity 0.8. Previously all would revert with `Vat/overflow`. In the few places where we still need an explicit overflow I think it makes more sense to keep all the reverts of this nature still the same which has to be the arithmetic code to match the others.

Reply:
Ok yeah, you are right, it makes sense to use the same error than the built in safe math.